src/azul/deployment.py

```diff
@@ -57,6 +57,10 @@ def apigateway(self):
     def account(self):
         return self.sts.get_caller_identity()['Account']
 
+    @cachedproperty
+    def user(self):
+        return self.sts.get_caller_identity()['UserId']
```

Comment:
Ah, I see that `AZUL_OWNER` is already documented for this purpose in [environment.py]. Good catch, thanks.

Reply:
I see. I haven't been able to find another way to programmatically retrieve the calling user's email address with the AWS API.

I implemented it this way because the original issue explicitly referenced calling `aws sts get-caller-identity`. `config.owner` is already set in the `Owner` tag. Should I drop the `Creator` tag entirely?

Reply:
I kind of think so. We should probably get @hannes-ucsc to weigh in here.